support/cas-server-support-pac4j-webflow/src/main/java/org/apereo/cas/web/flow/actions/DelegatedClientAuthenticationAction.java

```diff
@@ -135,6 +136,9 @@ protected Event doExecuteInternal(final RequestContext context) {
                 }
                 return finalizeDelegatedClientAuthentication(context, clientCredential.get());
             }
+            else if(StringUtils.isNotBlank(clientName)) {
```

Comment:
Please format the block correctly, and make sure you explain exactly why an authentication exception is being thrown.